python/cudf/cudf/tests/test_doctests.py

```diff
@@ -80,6 +80,16 @@ def chdir_to_tmp_path(cls, tmp_path):
         yield
         os.chdir(original_directory)
 
+    @pytest.fixture(autouse=True)
+    def prinoptions(cls):
+        # TODO: NumPy now prints scalars as `np.int8(1)`, etc. this should
+        #       be adapted evantually.
+        if np.lib.NumpyVersion(np.__version__) >= "2.0.0rc1":
```

Comment:
Another suggestion to use `packaging.version.parse` if you recommend against `np.lib` usage

Reply:
Since it is used in many tests, adopted now (and rebased, but all except last commits are unchanged).